introspection-engine/connectors/sql-introspection-connector/src/pair/default.rs

```diff
@@ -123,6 +124,7 @@ impl<'a> DefaultValuePair<'a> {
             (None, sql::ColumnTypeFamily::String) => match self.previous {
                 Some(previous) if previous.is_cuid() => Some(DefaultKind::Cuid),
                 Some(previous) if previous.is_uuid() => Some(DefaultKind::Uuid),
+                Some(previous) if previous.is_nanoid() => Some(DefaultKind::Nanoid(None)),
```

Comment:
What happens if the PSL schema has `@default(nanoid(2))`, and we introspect. Is the length removed and we get `nanoid()` as a result?

Reply:
That... is a great question. At first glance it does seem like that would be the case. I will figure that out and address the problem if necessary

Reply:
@pimeys I added the `nanoid` re-introspection to one of the introspection engine tests, and fixed the issue. let me know what you think!